parlai/agents/rag/retrievers.py

```diff
@@ -1125,11 +1131,14 @@ def pick_chunk(self, query, doc_text):
             # When there is no search query for the context
             return [("", 0)]
         tokens = self.text2tokens(doc_text)
-        doc_chunks = [
-            self.tokens2text(tokens[i : i + self.len_chunk])
-            for i in range(0, len(tokens), self.len_chunk)
-        ]
-        return self.chunk_reranker.get_top_chunks(query, doc_chunks)
+        if self.chunk_reranker != 'woi_chunk_retrieved_docs':
```

Comment:
Shouldn't this be the `chunk_ranker_type`? I think you are comparing the string to the `ChunkRanker` object here.

Reply:
oh yes you're right